package.json

```diff
@@ -13,6 +13,11 @@
     "babel-plugin-add-module-exports": "^1.0.2",
     "cross-env": "^5.2.0",
     "css-loader": "^2.1.1",
+    "dotenv": "^8.0.0",
+    "eslint": "^6.1.0",
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-plugin-prettier": "^3.1.0",
+    "eslint-plugin-svelte3": "^2.7.3",
```

Comment:
Can you follow instructions here? Existing .eslintrc file isn't enough and it doesn't lint svelte files.

https://www.npmjs.com/package/eslint-plugin-svelte3